src/index.js

```diff
@@ -2,7 +2,11 @@
 import Adapter from './Adapter';
 import Gun from 'gun/gun';
 
-Gun.on('opt', (context) => {
+Gun.on('opt', function(context) {
```

Comment:
haha! ;) the linter at work is just the opposite so now I'm in that habit. I fixed it up and the build passed.